jenkins_pipelines/environments/salt-shaker/manager-salt-shaker-products-next-sles16-bundle

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
+
+node('salt-shaker-tests') {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '20', daysToKeepStr: '4', artifactNumToKeepStr: '3')),
+        disableConcurrentBuilds(),
+        pipelineTriggers([
+            URLTrigger(
+                cronTabSpec: '* * * * *',
+                triggerLabel: "salt-shaker-tests",
+                labelRestriction: true,
+                entries: [URLTriggerEntry(
+                    url: 'https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/next:/testsuite/SLFO/repodata/repomd.xml',
```

Comment:
Now that `SLFO:1.2` was created for SLE16/SLMicro6.2, we should probably use it instead of `SLFO:Main`.